frontend/public/css/style.css

```diff
@@ -182,6 +182,7 @@ a:hover {
   --bs-btn-hover-border-color: var(--newblue);
   --bs-btn-active-bg: var(--newblue);
   --bs-btn-active-border-color: var(--newblue);
+  --bs-btn-disabled-bg: var(--gray-6);
```

Comment:
There is still a weird blue border around the button but otherwise the PR looks good to me.

Reply:
![Screenshot_2025-06-16_17-49-21](https://github.com/user-attachments/assets/a18db09f-efc2-4a0d-92ca-0cd11f8301df)

Reply:
How about this:
![image](https://github.com/user-attachments/assets/90b1a786-414c-4d18-be63-254506d1fd96)